pom.xml

```diff
@@ -150,15 +150,16 @@
 					<version>3.2.0</version>
 				</plugin>
 
-				<plugin>
+					<plugin>
 					<groupId>org.apache.maven.plugins</groupId>
 					<artifactId>maven-checkstyle-plugin</artifactId>
-					<version>${checkstyle.plugin.version}</version>
```

Comment:
This error is displayed: (Failed to execute goal org.apache.maven.plugins: maven-checkstyle-plugin: 2.17: check (validate) on project opennlp: Execution validate of goal org.apache.maven.plugins: maven-checkstyle- Plugin: 2.17: check failed: Plugin org.apache.maven.plugins: maven-checkstyle-plugin: 2.17 or one of its dependencies could not be resolved);

I'm not sure why, but it does not identify the checkstyle version this way. I'm using the latest version of maven and opennlp, with no modification to the pom, other than the dependency indicated in the Readme.

I just used mvn -DskipTests clean install, and the error appeared, I did the same by changing the pom, but there were other errors in opennlp tools, which did not complete the procedure. Exactly the same error that was displayed there in the pr test.